virt-v2v/cold/entrypoint

```diff
@@ -1,23 +1,66 @@
 #!/usr/bin/env bash
+shopt -s nullglob
 
-echo "Run virt-v2v with the following input:"
-cat /mnt/v2v/input.xml
-
-virt-v2v -v -x -i libvirtxml -o null --debug-overlays --no-copy --root=first /mnt/v2v/input.xml
-[ $? != 0 ] && exit 1
-
-echo "Conversion successful. Committing all overlays to local disks."
-for OVERLAY in /var/tmp/*.qcow2
-do
-	if ! qemu-img commit -p "$OVERLAY"
-	then
-		echo Failed to commit overlay "$OVERLAY"!
-		echo Unable to complete import!
-		exit 1
-	fi
+echo "Preparing virt-v2v"
+
+# This variable is used to build the list of arguments for virt-v2v.
+args=()
+
+# Temporary storage location
+DIR="/tmp/v2v"
```

Comment:
i think we had troubles with just `/tmp` in the past. maybe worth to changing it to `/var/tmp`?

Reply:
we did but with storing kubernetes stuff there (https://github.com/kubev2v/forkliftci/pull/37) - here it's within the container image

Reply:
regardless we also saw it with ovirt using OVAs. i think it's better to be safe than sorry.

Reply:
ah that's what you had in mind.. I don't think what virt-v2v does with that space is as complicated as inspecting an OVA source that is placed there so it would be fine but anyway, +1 to move it elsewhere if it doesn't work :)